tests/test_deployment/test_tasks.py

```diff
@@ -0,0 +1,127 @@
+import os
+
+import flask_login
+import pytest
+from flux import current_timeline
+
+from flask_app.blueprints.auth import get_or_create_user
+from flask_app.combadge import RemoteCombadge
+from flask_app.models import Beam, User, db
+from flask_app.tasks import beam_up, queue
+from flask_app.utils import RemoteHost
+
+OS_TYPES = ['linux', 'windows']
+HOST_NAME_BY_OS_TYPE = {
```

Comment:
So we statically assume that `gdc-qa-io-350` will always be there, being a Windows and available host? Why that? (Same regarding 017)

Reply:
These tests aren't run by the CI, they can't be run by the CI because they use internal resources/tools

Reply:
I made them into env vars